engine/engine/src/engine.cpp

```diff
@@ -539,7 +579,11 @@ namespace dmEngine
         char project_file[DMPATH_MAX_PATH];
         char content_root[DMPATH_MAX_PATH] = ".";
```

Comment:
Since we now have _two_ paths that are a bit confusing, I think the "content_root" should be renamed.
E.g. how do they differ, and _when_ do they differ.

Reply:
I fixed that.